release/README.md

```diff
@@ -8,22 +8,55 @@ Template configurtion file for the scripts found in this directory.
 Copy to ~/.server.config and fill out with the appropriate values for
 services provided on the server running these automation scripts.
 
+---
+
+## Automated Publishing Scripts
+
+The `crontab` configuration driving this automation:
+```
+03 */2 * * * bin/release/mk-release-snapshots.sh
+21 */2 * * * bin/release/mk-cfgman-docs.sh
+23 */2 * * * bin/release/mk-langpack.sh
+25 */2 * * * bin/release/mk-www-manuals.sh
```

Comment:
If this is what we have now, then this question is out of this PR scope, but I would appreciate an answer anyway: Why stagger start times but potentially run these scripts in parallel instead of doing something simple like

```
 03 */2 * * * bin/release/mk-all.sh
```

where bin/release/mk-all.sh just runs all the necessary scripts?

Reply:
These used to be sections of a giant do-everything script. It took a long time to sequentially perform all the tasks and would overlap with itself occasionally when there was network connectivity lag, or lots of file updates to sync.

This adds enough parallelization to solve that issue without too much complexity. Making sure the releases happen on time.

Reply:
Understood, thank you.